yarn-project/simulator/src/avm/opcodes/memory.ts

```diff
@@ -218,7 +218,7 @@ export class CalldataCopy extends Instruction {
   }
 
   protected override gasCost(memoryOps: Partial<MemoryOperations & { indirect: number }> = {}) {
-    const baseGasCost = mulGas(getBaseGasCost(this.opcode), this.copySize);
```

Comment:
Was this the only instance of scaling base cost?

Reply:
SSTORE is the other one I found

Reply:
Ah, I see. Those will get solved once we do this: https://github.com/AztecProtocol/aztec-packages/pull/5153